interface/src/app/tribe_client/geneset-search-bar.tpl.html

```diff
@@ -0,0 +1,12 @@
+<div>
+  <form class="form-inline">
+    <label>Organism:</label>
+    <select ng-disabled="true">
+      <option ng-selected="true" value="organism">{{organism}}</option>  
+    </select>
+
+    <input ng-model="search.query" placeholder="Search text">
+
+    <button class="btn btn-default" ng-click="searchGenesets(search)">Search</button>
```

Comment:
This line invokes `searchGenesets(search)`, but have a look at the implementation of that function... ☆